Add stop command and alias to README (#1051)
vivid-planet/comet: README.md

-   [Docker Desktop](https://www.docker.com/products/docker-desktop/)
-   [nvm](https://github.com/nvm-sh/nvm)

> The Node.js process manager "dev-process-manager" (https://github.com/vivid-planet/dev-process-manager) is used in the local development environment.

Reviewer: ```suggestion
[dev-process-manager](https://github.com/vivid-planet/dev-process-manager) is used for local development.
```

I'd move this clarification into the "Start development processes" section

Storybook will be available at [http://localhost:26638/](http://localhost:26638/)

### Stop Processes

```bash
npx dev-pm shutdown

> ```

Reviewer 2: It is not clear that dev-pm is used, so I would state its usage somewhere above, so this makes sense

### Handy Alias

```
alias dpm="npm exec -- dev-pm"
```

> 

Reviewer 2: the alias is already documented here: https://github.com/vivid-planet/dev-process-manager, so I would remove this
